rules-tests/CodeQuality/Rector/FuncCall/SortNamedParamRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\FuncCall\SortNamedParamRector\Fixture;
+
+function run($foo = null, $bar = null, $baz = null) {}
+
+run(bar: $bar, foo: $foo);
+
+run($foo, baz: $baz, bar: $bar);
+
+run(...);
```

Comment:
I prefer to have multiple fixture files for this, eg:

- `fixture.php.inc`
- `skip_sorted.php.inc`
- `skip_first_class_callable.php.i.nc`

so easier to debug when there is a issue per target fixture.

Reply:
Done